config_defaults_inc.php

```diff
@@ -3498,14 +3498,28 @@
 
 /**
  * Add custom options to the main menu.  For example:
+ *
  * $g_main_menu_custom_options = array(
- *     array( "My Link",  MANAGER,       'my_link.php' ),
- *     array( "My Link2", ADMINISTRATOR, 'my_link2.php' )
+ *     array( 
+ *         'title'        => 'My Link',
+ *         'access_level' => MANAGER,
+ *         'url'          => 'my_link.php',
+ *         'icon'         => 'fa-plug'
+ *     ),
+ *     array( 
+ *         'title'        => 'My Link2',
+ *         'access_level' => ADMINISTRATOR,
+ *         'url'          => 'my_link2.php',
+ *         'icon'         => 'fa-plug'
```

Comment:
In commit message aa5597ba20c3f4e52fbbc7ed65933d4611fc4494 you imply that the icon is optional and replaced by a default one; assuming this applies here as well, it would be good to document that, and also to describe what the default icon is.